packages/plugin-analytics/src/utils/PlausibleAnalytics.ts

```diff
@@ -0,0 +1,100 @@
+import { logger } from 'robo.js'
+import { BaseEngine, EventOptions, ViewOptions } from './analytics'
+
+export class PlausibleAnalytics extends BaseEngine {
+	private _PLAUSIBLE_DOMAIN = process.env.PLAUSIBLE_DOMAIN
```

Comment:
Please use WavePlay's naming convention for class instances here.

```ts
private _domain = process.env.PLAUSIBLE_DOMAIN
```

May also be nice to be able to customize this in the constructor.